From e549f2c34279f91b2f5043f0b1498d13711c36db Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 15 Oct 2025 19:04:38 +0200
Subject: [PATCH] Don't move the frame offscreen for static windows with 0,0

Some clients like Qt 6.10 sends a static position to the top left corner.
Moving the position above that causes the title and border to be placed
out of screen.
---
 src/wmmgr.cc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/wmmgr.cc b/src/wmmgr.cc
index 68cff75e1..353403b9f 100644
--- a/src/wmmgr.cc
+++ b/src/wmmgr.cc
@@ -1724,10 +1724,6 @@ void YWindowManager::placeWindow(YFrameWindow *frame,
             posX -= 2 * borderWidth - client->getBorder() - 1;
         if (gy > 0)
             posY -= 2 * borderHeight + titleHeight - client->getBorder() - 1;
-        if (gx == 0 && gy == 0 && client->winGravity() == StaticGravity) {
-            posX -= borderWidth;
-            posY -= borderHeight + titleHeight;
-        }
         posY -= borderOffset;
     }
 
